beetsplug/mbsync.py

```diff
@@ -82,18 +84,29 @@ def singletons(self, lib, query, move, pretend, write):
                                item_formatted)
                 continue
 
-            # Get the MusicBrainz recording info.
-            track_info = hooks.track_for_mbid(item.mb_trackid)
-            if not track_info:
-                self._log.info(u'Recording ID not found: {0} for track {0}',
-                               item.mb_trackid,
-                               item_formatted)
-                continue
+            valid_trackid = re.match(
+                     r"(\d|\w){8}-(\d|\w){4}-(\d|\w){4}-(\d|\w){4}-(\d|\w){12}",
+                     item.mb_trackid)
+
+            # Do we have a valid MusicBrainz TrackId
+            if valid_trackid:
```

Comment:
In order to mimic the style of the next check in this function, what do you think about using `continue` to about early instead of indenting the rest of the loop body under the `if` block? This makes it easy to see that the check aborts the process and more clearly associates the error message with its corresponding check.